src/main/java/com/gimranov/zandy/app/api/ZoteroService.java

```diff
@@ -0,0 +1,25 @@
+package com.gimranov.zandy.app.api;
+
+import com.gimranov.zandy.app.model.Collection;
+import com.gimranov.zandy.app.model.Item;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.Headers;
+import retrofit2.http.Path;
+
+public interface ZoteroService {
+    @Headers("Zotero-API-Version: 3")
```

Comment:
I think you can pull the header out into an interceptor that we use for all the requests, which will save some time

There is also https://github.com/avram/libzotero-java which covers some of this in an attempt to do a nice adapter for all Java applications; I didn't finish that but my intent had been to write a solid Java client then have the Android app use that client.